.github/workflows/build_and_test.yml

```diff
@@ -0,0 +1,95 @@
+# This workflow will build a Java project with Gradle
```

Comment:
build_and_test seems a little redundant. maybe just test.yml? (I know that's what we have on CRL atm, but doesn't seem like a great filename).
also, it would be nice if git tracked the history of this file from unit_test.yml, if that's not too much work. not necessary.

Reply:
changed to test.yml.
I deleted the file and tried use git mv command, but it still can not get the git history. What would be the command to let git track the change history? Thanks

Reply:
probably not worth it then. If git mv doesn't do it, not sure what else I'd try.